.bin/test.ts

```diff
@@ -0,0 +1,37 @@
+import { assert } from '@japa/assert'
+import { specReporter } from '@japa/spec-reporter'
+import { runFailedTests } from '@japa/run-failed-tests'
+import { processCliArgs, configure, run } from '@japa/runner'
+
+/*
+|--------------------------------------------------------------------------
+| Configure tests
+|--------------------------------------------------------------------------
+|
+| The configure method accepts the configuration to configure the Japa
+| tests runner.
+|
+| The first method call "processCliArgs" process the command line arguments
+| and turns them into a config object. Using this method is not mandatory.
+|
+| Please consult japa.dev/runner-config for the config docs.
+*/
+configure({
+  ...processCliArgs(process.argv.slice(2)),
+  ...{
+    files: ['test/**/*.spec.ts'],
+    plugins: [assert()],
+    reporters: [specReporter(), runFailedTests()],
```

Comment:
`runFailedTests` should be under `plugins` array :)

Reply:
Oops